poh/src/poh_recorder.rs

```diff
@@ -462,9 +501,8 @@ impl PohRecorder {
                 working_bank.max_tick_height,
                 working_bank.bank.slot()
             );
-            let working_slot =
-                (working_bank.max_tick_height / self.ticks_per_slot).saturating_sub(1);
-            self.start_slot = working_slot;
+            self.start_bank = working_bank.bank.clone();
+            let working_slot = self.start_slot();
```

Comment:
This is equivalent to the original code because:

1) `working_bank` is only ever set by `self.set_bank()`, in which `working_bank.max_tick_height` is set to `working_bank.bank.max_tick_height()`: 
https://github.com/solana-labs/solana/blob/857fce13f9fc879b8811cefacb4939dab714781f/poh/src/poh_recorder.rs#L412

2) This means because `bank.max_tick_height == (bank.slot() + 1) * ticks_per_slot`: https://github.com/solana-labs/solana/blob/master/runtime/src/bank.rs#L1282, then the old code 

```
(working_bank.max_tick_height / self.ticks_per_slot).saturating_sub(1);
```

is exactly equivalent to `working_bank.bank.slot()`